submission/submission-webapp/src/itest/resources/application.yml

```diff
@@ -18,7 +18,9 @@ spring:
     port: 5672
 
 app:
-  basepath:
+  submissionPath:
+  ftpPath:
+  magicPath:
```

Comment:
Don't we need this property in the application's main config file?

Also, I didn't see it's being used in the code, only in the `ApplicationProperties` class but I assume it's the user magic folder? In any case, I think the property name can be improved or at least add a comment indicating what it's used for.

Reply:
ohh yes let me remove this one, is for future changes